### ISSUE TITLE
Undeclared dependency Test2::Tools::Explain

### ISSUE THREAD

**Issue by eserte** [cpanel/Test2-Harness-Renderer-JUnit#11]
t/test.t fails if Test2::Tools::Explain is not installed:
```
Can't locate Test2/Tools/Explain.pm in @INC (you may need to install the Test2::Tools::Explain module) (@INC contains: ...  .) at t/test.t line 17.
BEGIN failed--compilation aborted at t/test.t line 17.
t/test.t .......... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by eserte** [comment]
Another possibly undeclared dependency: XML::Simple:
```
Can't locate XML/Simple.pm in @INC (you may need to install the XML::Simple module) (@INC contains: ...  .) at t/test.t line 20.
BEGIN failed--compilation aborted at t/test.t line 20.
t/test.t .......... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by toddr** [comment]
Thanks. Just noticed it this morning. Should be fixed in eac30096f0288a4c9d56cfa9ad99620858eda772
